src/modules/app/constants/buildConfig.js

```diff
@@ -2,7 +2,32 @@
 
 import type { BuildConfigType } from '../../../../build-configs/configs/BuildConfigType'
 import loadBuildConfig from '../../../../build-configs'
+import { INTEGREAT_ICONS } from '../../../../build-configs/configs/integreat'
+import { MALTE_ICONS } from '../../../../build-configs/configs/malte'
+import integreatAppLogo from '../../../../build-configs/configs/integreat/assets/app-logo.png'
+import integreatLocationMarker from '../../../../build-configs/configs/integreat/assets/location-marker.svg'
+import malteAppLogo from '../../../../build-configs/configs/malte/assets/app-logo.svg'
+import malteLocationMarker from '../../../../build-configs/configs/malte/assets/location-marker.svg'
 
-const buildConfig = (): BuildConfigType => loadBuildConfig(process.env.BUILD_CONFIG_NAME)
+const buildConfigIcons = (iconSet: string) => {
```

Comment:
I don't quite get why you parse the itemset here instead of directly parsing th MalteBuildConfig. Is it because of the test maybe? Or is there any other reason?

Reply:
You mean why I don't include the icons directly in the build config? Importing .svg files does not work with plain js. Also, we would need to handle .svgs and .pngs separately upon creating xcconfig files since these are simple key/value files. I tried it anyway, but couln't get it to work, maybe because the svg plugin we are using (https://github.com/aeirola/react-native-svg-asset-plugin) does not work for files outside the src dir.

Reply:
Any ideas would be appreciated though :)

Reply:
ah. okay. that's annoying ^^ . Then I'd say leave it as it is, even though I still feel it is kind of out of place here. Maybe @maxammann  has a plan

Reply:
Yep, maybe also @Schedulaar ?

Reply:
Something that should work according to the README of the react-native-svg-asset-plugin is to use the `require` call.

So include the following in the build config:
```
iconPaths: {
  appLogo: './build-configs/configs/integreat/assets/app-logo.png' // path relative from project root
}
```

Then in the code you can use `require('../../../../' + buildConfig.iconPaths.appLogo)`

Reply:
The Build Config only can contain primives: bools, arrays, maps, strings, numbers etc.
Also the config should only import js files as it needs to work with only nodejs.

An improvment to that way would be to have a function: `loadIcons(buildConfig)` which returns an object of paths to the PNGs of react-native-svg-asset-plugin.

Reply:
Not sure if that is actually "prettier" than using the flag. Surely it is more extensible, but handling Icons feels all over the place like this...

Reply:
As discussed the suggestion by @maxammann does not work with the plugin. Leaving it as it is.